SergeiDemyanenko/PlatformaticaQA
# Reference values: Input ( TC-1 New record )

https://trello.com/c/9464XQBY/154-entity-reference-valuesawdawd-input-tc-1-new-record-positive

Test steps:
Steps:
1. Click on Reference values in side menu.
2. Click on add button.
3. Enter Label = 'Label for new record',
4. Enter Filter 1 = 'Filter Test 1'
5. Enter Filter 2 = 'Filter Test 2'
6. Click on 'Save' button

Expected result:
- Created record displayed in list
- Data in record same as was entered
